src/validations/access/common/profiles.ts

```diff
@@ -0,0 +1,34 @@
+import { EthAddress } from '@dcl/schemas'
+import {
+  ContentValidatorComponents,
+  DeploymentToValidate,
+  OK,
+  ValidateFn,
+  validationFailed,
+  ValidationResponse
+} from '../../../types'
+
+export function createPointerValidateFn(components: Pick<ContentValidatorComponents, 'externalCalls'>): ValidateFn {
```

Comment:
The names of the functions exported in `common/*` are the same exported in `../validations/*`. Maybe it would be good to rename the functions in `common/*` to `createPointerAccessValidateFn` or whatever